region/maridia/outer.json

```diff
@@ -4626,7 +4865,14 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateUnderwater",
-                    "canSpringwall"
+                    "canSpringwall",
+                    "canResetFallSpeed"
```

Comment:
I'm sure there are many places to add `canResetFallSpeed` following a springball jump or HBJ.
Here, it can be done without the unmorph but its a little tricky to do the jump well enough.  Or you could turn springball off again and springfling.
You can also reach the item with springballjump without a walljump 🤔 Even land on the pedestal with a springfling.